test/lit/passes/optimize-casts.wast

```diff
@@ -274,9 +284,21 @@
     (drop
       (local.get $x)
     )
+    (local.set $x
+      ;; Cannot look through for sets at the moment
```

Comment:
Generally we try to make comments full sentences, including `.`, although we're not perfect about it.
```suggestion
      ;; Cannot look through for sets at the moment.
```